pkg/cmd/driftctl.go

```diff
@@ -54,6 +61,7 @@ func NewDriftctlCmd(build build.BuildInterface) *DriftctlCmd {
 
 	cmd.PersistentFlags().BoolP("help", "h", false, "Display help for command")
 	cmd.PersistentFlags().BoolP("no-version-check", "", false, "Disable the version check")
+	cmd.PersistentFlags().BoolP("error-reporting", "", false, "Enable error reporting.\nWARNING: may leak some sensitive data")
```

Comment:
Should be `may leak sensitive data` since data is invariable

Reply:
Fixed